scripts/e2e

```diff
@@ -2,11 +2,13 @@
 # Reinstall all canisters and run E2E tests in your local environment.
 (
     dfx canister create --all &&
-    dfx deploy evm_rpc --argument "(record {nodesInSubnet = 13})" --mode reinstall -y &&
-    dfx deploy evm_rpc_fiduciary --argument "(record {nodesInSubnet = 28})" --mode reinstall -y &&
-    dfx deploy e2e_rust --mode reinstall -y &&
+    dfx deploy evm_rpc --argument "(record {nodesInSubnet = 28})" --mode reinstall -y &&
+    dfx deploy evm_rpc_staging_13_node --argument "(record {nodesInSubnet = 13})" --mode reinstall -y &&
+    dfx deploy evm_rpc_staging_fiduciary --argument "(record {nodesInSubnet = 28})" --mode reinstall -y &&
+    npm run generate &&
+    dfx deploy e2e_rust &&
     dfx canister call e2e_rust test &&
-    dfx deploy e2e_motoko --mode reinstall -y &&
+    dfx deploy e2e_motoko &&
```

Comment:
I came across what seems like a heisenbug or corner case with `dfx deploy` where using `--mode reinstall` on the first deployment causes a mysterious `ic-cdk-bindgen` error message. This occasionally happened on CI and other times on my local machine, but it was unclear what was specifically causing this. Just wanted to quickly mention this somewhere, since it took some trial and error to narrow down the issue.

Here's the error message for posterity:

```
thread 'main' panicked at 'Cannot find candid path: NotPresent'
```

The solution was to remove the `--mode reinstall` flag from `dfx deploy`.

Reply:
Heisenbug.   :smile: 

Is this something we should bring up with the SDK team?